GameServer/commands/gmcommands/area.cs

```diff
@@ -85,57 +110,86 @@ public void OnCommand(GameClient client, string[] args)
+						// Message: [INFO] Description:
+						ChatUtil.SendTypeMessage((int)eMsg.Debug, client, "GMCommands.Area.Msg.AreaDesc", null);
+						// Message: --- Name: {0}
+						ChatUtil.SendTypeMessage((int)eMsg.Debug, client, "GMCommands.Area.Msg.AreaName", area.Description);
+						// Message: --- X: {0}, Y: {1}, Z: {2}, Region: {3}
+						ChatUtil.SendTypeMessage((int)eMsg.Debug, client, "GMCommands.Area.Msg.AreaLoc", area.X, area.Y, area.Z, area.Region);
+						// Message: --- Radius: {0}
+						ChatUtil.SendTypeMessage((int)eMsg.Debug, client, "GMCommands.Area.Msg.AreaRadius", area.Radius);
+						// Message: --- Broadcast: {0}
+						ChatUtil.SendTypeMessage((int)eMsg.Debug, client, "GMCommands.Area.Msg.AreaBroad", area.CanBroadcast.ToString());
+						// Message: --- Sound: {0}
+						ChatUtil.SendTypeMessage((int)eMsg.Debug, client, "GMCommands.Area.Msg.AreaSound", area.Sound);
+
 						break;
 					}
 				#endregion Create
+				#region Info
+				// --------------------------------------------------------------------------------
+				// INFO (Admin/GM command)
+				// '/area info'
+				// Displays relevant information regarding the areas in which the client are located.
+				// --------------------------------------------------------------------------------
 				case "info":
                     {
-						string name = "(Area Info)";
+						string name = "Area Information";
 						var info = new List<string>();
 						//info.Add("        Current Areas : " + client.Player.CurrentAreas);
 						//info.Add(" ");
 						var areas = client.Player.CurrentAreas;
+
                         foreach (var area in areas)
                         {
-							info.Add("Area ClassType: " + area.GetType());
+	                        info.Add(" ");
+							info.Add("Area Class Type: " + area.GetType());
 							info.Add(" ");
-							info.Add("Area ToString: " + area.ToString());
+							info.Add("Area String: " + area.ToString());
 							info.Add(" ");
 
-							if(area is KeepArea ka)
+							if (area is KeepArea ka)
                             {
 								info.Add("Area Keep: " + ka.Keep.Name);
 								info.Add(" ");
 
                                 foreach (var guard in ka.Keep.Guards.Values)
                                 {
-									info.Add("Area Guard: " + guard);
+									info.Add("Keep Guard: " + guard);
 									info.Add(" ");
 								}
 
                                 foreach (var component in ka.Keep.KeepComponents)
                                 {
-									info.Add("Area Component: " + component);
+									info.Add("Keep Component: " + component);
 									info.Add(" ");
-								}
-								
-							}
+                                }
+                            }
 
-							
-						}
+							info.Add("AllCommands.Header.Note.Dashes");
+                        }
+
+                        ChatUtil.SendWindowMessage((int)eWindow.Text, client, name, info);

```

Comment:
System.InvalidCastException: Unable to cast object of type 'System.Collections.Generic.List`1[System.String]' to type 'System.String'.
   at DOL.GS.ChatUtil.SendWindowMessage(Int32 type, GameClient target, String title, Object[] args)
   at DOL.GS.Commands.AreaCommandHandler.OnCommand(GameClient client, String[] args)
   at DOL.GS.ScriptMgr.ExecuteCommand(GameClient client, GameCommand myCommand, String[] pars)
   at DOL.GS.ScriptMgr.HandleCommand(GameClient client, String cmdLine)